src/content/contributing/release-process/_index.en.md

```diff
@@ -4,154 +4,209 @@ date: 2020-03-18T16:03:26+01:00
+case, the suggested naming convention is to append `-preN`.
 
-<!-- TODO(mangelajo) https://github.com/submariner-io/submariner-website/issues/46 -->
+## Create Submariner product release
 
+The following outlines the steps to be taken in order to create a full Submariner product release. As an example, we'll
+use version `v0.5.0`.
 
-## Step 2: Create a Lighthouse Release
+Project releases can either be created via the git CLI or the GitHub UI. This guide uses the GitHub UI as it is
+simpler:
 
-To create lighthouse release artifacts follow the steps below.
+1) Navigate to the project's main releases page.
 
-### Build Lighthouse Agent and CoreDNS
+2) Click `Draft a new release`
 
-Assuming that you have an existing lighthouse git directory, run the following steps .
+3) Fill in the `Tag version` field with the new version.
 
-```bash
-cd lighthouse
-git stash
-git remote add upstream ssh://git@github.com/submariner-io/lighthouse
-git fetch -a -v -t upstream
-git checkout remotes/upstream/master -B master
-git tag -s -m "Globalnet Overlapping IP support RC0" v0.2.0-rc0
-git push upstream v0.2.0-rc0
-```
+4) Fill in the `Release title` field as appropriate. Typically this is just the version (sans the leading `v`) as a release
+   usually contains various changes. But if it's a targeted release then the `Release title` can reflect that.
+   
+5) Enter information to describe the release. This is optional for pre-releases but should be filled in for
+   a final release.
+   
+6) If this is not a final release, mark the checkbox `This is a pre-release`.
 
-A tagged release should appear [here](https://github.com/submariner-io/lighthouse/tags)
+7) Click `Publish release`.
 
-> https://github.com/submariner-io/lighthouse/tags
+For most projects a GitHub action job will be initiated to build release artifacts and publish to quay. This will take
+several minutes. You can monitor the progress from the project's main page. A small yellow circle should be present to
+the right of the heading above the file listing which indicates it's in progress. You can click it to see details. When
+complete, it will change to either a green check mark or a red X.
 
-A build for v0.2.0-rc0 should start and appear under the "Active branches" section [here](https://travis-ci.com/github/submariner-io/lighthouse/branches)
+### Step 0: Create a `subctl` pre-release, optional
 
-> https://travis-ci.com/github/submariner-io/lighthouse/branches
+Since `subctl` is provided by `shipyard`'s dapper base image, if there are specific changes to `subctl` which other
+projects require, you will need to create a pre-release (for example `v0.5.0-pre0`) of the `submariner-operator` project: 
 
-For this example the build can be found [here](https://travis-ci.com/github/submariner-io/lighthouse/builds/153946391).
+1) Navigate to the [releases](https://github.com/submariner-io/submariner-operator/releases) page and create the release
```

Comment:
Might be good to extract this link to a reference, as it's used quite a few times.

Reply:
I assume you mean that "releases"  is used several times but the underlying URL is different.

Reply:
Oh I missed that they are to different projects' release pages, my bad. Okay, the way you have it is perfect then!